chia/cmds/farm_funcs.py

```diff
@@ -181,69 +181,81 @@ async def challenges(farmer_rpc_port: int, limit: int) -> None:
         )
 
 
-async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int, farmer_rpc_port: int) -> None:
+async def summary(
+        rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int, farmer_rpc_port: int, json_format: bool
+) -> None:
     amounts = await get_wallets_stats(wallet_rpc_port)
     plots = await get_plots(harvester_rpc_port)
     blockchain_state = await get_blockchain_state(rpc_port)
     farmer_running = await is_farmer_running(farmer_rpc_port)
 
-    print("Farming status: ", end="")
+    summary = {}
```

Comment:
Python is probably forgiving but I'm not sure that this should be called the same as the function using it.

Reply:
Sure, that makes sense. Anyway - naming is the hardest part of software engineering, so having in mind that this function is probably never going to be called recursively shadowing its name should be fine as well.

Reply:
Everything else uses the word show.

Reply:
Fair point, I'll change the name of function to `show_summary`